mars/dataframe/expressions/core.py

```diff
@@ -54,6 +90,19 @@ def new_dataframe(self, inputs, shape=None, dtypes=None, index_value=None, colum
         return self.new_dataframes(inputs, shape=shape, dtypes=dtypes,
                                    index_value=index_value, columns_value=columns_value, **kw)[0]
 
+    def new_seriess(self, inputs, shape=None, dtype=None, index_value=None, name=None,
```

Comment:
Looks a little strange for me. Maybe we can have a better name given that ``series`` itself is a plural form.

Reply:
`series` is already plural, use `seriess` to keep consistency with others.